dll/win32/shell32/folders/CRecycleBin.cpp

```diff
@@ -1125,7 +1125,7 @@ HRESULT WINAPI SHEmptyRecycleBinW(HWND hwnd, LPCWSTR pszRootPath, DWORD dwFlags)
         ret = RegGetValueW(HKEY_CURRENT_USER,
                            L"AppEvents\\Schemes\\Apps\\Explorer\\EmptyRecycleBin\\.Current",
                            NULL,
-                           RRF_RT_REG_SZ,
+                           RRF_RT_REG_EXPAND_SZ | RRF_NOEXPAND,
```

Comment:
And what if the value was REG_SZ instead? And who is going to expand the string by the way?

Reply:
Hmm, perhaps I also should consider this case.

Reply:
As MSDN says, any of these flags can be just combined if we need to retrieve more than one type of data, so I think if to specify both `RRF_RT_REG_SZ` and `RRF_RT_REG_EXPAND_SZ`, it will search for the value of both of these types.

Reply:
Fixed, thanks @HBelusca.